.pre-commit-config.yaml

```diff
@@ -268,7 +268,7 @@ repos:
     # v0.12.0 requires ruby2.7, however openSUSE Leap 15.4 officially
     # only provides ruby2.5 which is also required for texlive
     # => stay at v0.11.0
-    rev: "v0.11.0"
+    rev: "v0.12.0"
```

Comment:
markdownlint must be downgraded to version 0.9.0, because higher versions require ruby2.6 or higher, but openSUSE Leap 15.4 officially only provides ruby2.5.